ivc/src/ivc/interpreter.rs

```diff
@@ -923,6 +1017,10 @@ where
         let sel = env.read_column(IVCColumn::BlockSel(i));
         env.assert_zero(sel.clone() * (sel.clone() - Env::constant(F::one())));
     }
+
+    // For now iteration is only 0 or 1
```

Comment:
Not adding it in https://github.com/o1-labs/proof-systems/pull/2372/files. We don't want this constraint as it won't be the case that fold_iteration will be 1 or 0.